olive/passes/onnx/model_builder.py

```diff
@@ -36,6 +36,15 @@ class Precision(str, Enum):
         def __str__(self) -> str:
             return self.value
 
+    class AccuracyLevel(int, Enum):
+        fp32 = 1
+        fp16 = 2
+        bf16 = 3
+        int8 = 4
+
+        def __str__(self) -> str:
```

Comment:
do we need the `__str__` method since it is int enum?

Reply:
Prints a simpler representation rather than the full name.